packages/components/src/components/primitives/form/DropdownField/DropdownNativeSelect/DropdownNativeSelect.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { forwardRef, useState } from 'react';
+import cn from 'classnames';
+import { Icon } from 'components/primitives';
+import { expand_more as ExpandMore } from 'components/primitives/Icon/icons/navigation';
+import { getDropdownSingleSelectClasses } from 'utils/getDropdownSingleSelectClasses';
+
+export const DROPDOWN_NATIVE_DEFAULT_TEST_ID = 'dropdown_native';
+
+export interface Option {
+  label: any;
+  value: any;
```

Comment:
can we receive any value for those properties?, we should avoid using any whenever possible

Reply:
@tkanzakic I think we have to let any, right now we don’t know what type could be this, because I saw on abstract some label with icons. https://share.goabstract.com/1e3aa0aa-fba8-4b47-aede-7ceb8fb6baeb?mode=design&sha=c8a4e5c55a37e48682b5f36f87dc448ae1a876ca